core/metrics-spi/src/main/java/software/amazon/awssdk/metrics/metrics/SdkDefaultMetric.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.metrics.metrics;
+
+import static software.amazon.awssdk.metrics.MetricCategory.DEFAULT;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.metrics.MetricCategory;
+import software.amazon.awssdk.metrics.meter.Metric;
+import software.amazon.awssdk.utils.ToString;
+
+/**
+ * A enum representing the metrics supported by the SDK. All metrics collected by the SDK will be declared in this
+ * enum class along with the metric's corresponding {@link MetricCategory}.
+ */
+// TODO List is not complete
+@SdkPublicApi
+public enum SdkDefaultMetric implements Metric {
+
+    /**
+     * Service ID of the AWS service that the API request is made against
+     */
+    Service("Service", DEFAULT),
```

Comment:
This enum needs to be updated to use upper case

Reply:
Will update.